go/vt/vttest/util.go

```diff
@@ -0,0 +1,32 @@
+package vttest
```

Comment:
Let's rename this file to shard_name.go. 'util' has become like 'goto'. You're supposed to avoid it as much as possible :).